src/core/Global.ttslua

```diff
@@ -1,102 +1,78 @@
+TOKEN_DATA = {
+  damage   = {image = "http://cloud-3.steamusercontent.com/ugc/1758068501357115146/903D11AAE7BD5C254C8DC136E9202EE516289DEA/", scale = {0.17, 0.17, 0.17}},
+  horror   = {image = "http://cloud-3.steamusercontent.com/ugc/1758068501357163535/6D9E0756503664D65BDB384656AC6D4BD713F5FC/", scale = {0.17, 0.17, 0.17}},
+  resource = {image = "http://cloud-3.steamusercontent.com/ugc/1758068501357192910/11DDDC7EF621320962FDCF3AE3211D5EDC3D1573/", scale = {0.17, 0.17, 0.17}},
+  doom     = {image = "https://i.imgur.com/EoL7yaZ.png", scale = {0.17, 0.17, 0.17}},
+  clue     = {image = "http://cloud-3.steamusercontent.com/ugc/1758068501357164917/1D06F1DC4D6888B6F57124BD2AFE20D0B0DA15A8/", scale = {0.15, 0.15, 0.15}}
 }
 
---[[ The OnLoad function. This is called after everything in the game save finishes loading.
-Most of your script code goes here. --]]
-function onload()
-  --Player.White.changeColor('Yellow')
-  tokenplayerone = {
-    damageone = "http://cloud-3.steamusercontent.com/ugc/1758068501357115146/903D11AAE7BD5C254C8DC136E9202EE516289DEA/",
-    damagethree = "http://cloud-3.steamusercontent.com/ugc/1758068501357113055/8A45F27B2838FED09DEFE492C9C40DD82781613A/",
-    horrorone = "http://cloud-3.steamusercontent.com/ugc/1758068501357163535/6D9E0756503664D65BDB384656AC6D4BD713F5FC/",
-    horrorthree = "http://cloud-3.steamusercontent.com/ugc/1758068501357162977/E5D453CC14394519E004B4F8703FC425A7AE3D6C/",
-    resource = "http://cloud-3.steamusercontent.com/ugc/1758068501357192910/11DDDC7EF621320962FDCF3AE3211D5EDC3D1573/",
-    resourcethree = "https://i.imgur.com/1GZsDTt.png",
-    doom = "https://i.imgur.com/EoL7yaZ.png",
-    clue = "http://cloud-3.steamusercontent.com/ugc/1758068501357164917/1D06F1DC4D6888B6F57124BD2AFE20D0B0DA15A8/"
-  }
-
-  TOKEN_DATA = {
-    clue = {image = tokenplayerone.clue, scale = {0.15, 0.15, 0.15}},
-    resource = {image = tokenplayerone.resource, scale = {0.17, 0.17, 0.17}},
-    doom = {image = tokenplayerone.doom, scale = {0.17, 0.17, 0.17}},
-    damage = {image = tokenplayerone.damageone, scale = {0.17, 0.17, 0.17}},
-    horror = {image = tokenplayerone.horrorone, scale = {0.17, 0.17, 0.17}}
-  }
-
-  getObjectFromGUID("6161b4").interactable=false
-  getObjectFromGUID("721ba2").interactable=false
-  getObjectFromGUID("9f334f").interactable=false
-  getObjectFromGUID("23a43c").interactable=false
-  getObjectFromGUID("5450cc").interactable=false
-  getObjectFromGUID("463022").interactable=false
-  getObjectFromGUID("9487a4").interactable=false
-  getObjectFromGUID("91dd9b").interactable=false
-  getObjectFromGUID("f182ee").interactable=false
-  getObjectFromGUID("7bff34").interactable=false
-
-end
-
-function onObjectDrop(player, obj)
---  local mat = getObjectFromGUID("dsbd0ff4")
---  log(mat.positionToLocal(obj.getPosition()))
-end
-
-function take_callback(object_spawned, mat)
-    customObject = object_spawned.getCustomObject()
-    local player = mat.getGUID();
+IMAGE_TOKEN_MAP = {
```

Comment:
Rename to CHAOS_TOKEN_IMAGE_MAP

Reply:
this is referenced by a lot of cards (mostly the ones that have a context menu with "sealing" options - not changing the name for now